src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+}
```

Comment:
Don't reset margins for *, do it for tags directly, in this case you need it for body only